spec/migrations/20181001131632_add_conversion_host_id_to_miq_request_tasks_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_migration
+
+describe AddConversionHostIdToMiqRequestTasks do
+  let(:task_stub) { migration_stub(:MiqRequestTask) }
+  let(:host_stub) { migration_stub(:Host) }
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+
+  migration_context :up do
+    it "creates conversion host" do
+      host = host_stub.create!
+      task = task_stub.create!(
+        :type    => 'ServiceTemplateTransformationPlanTask',
+        :options => { :transformation_host_id => host.id }
+      )
+
+      migrate
+
+      task.reload
+      expect(task.options).to eq({})
```

Comment:
can you add another dummy key in this `options` field and here check only that `:transformation_host_id` is removed?

Reply:
Done.